models/alignn_ff/metadata.yml

```diff
@@ -0,0 +1,42 @@
+model_name: ALIGNN FF
+model_version: 2023.07.01
+matbench_discovery_version: 1.0
+date_added: "2023-07-11"
+date_published: "2022-09-16"
+authors:
+  - name: Kamal Choudhary
+    affiliation: National Institute of Standards and Technology
+    email: kamal.choudhary@nist.gov
+    orcid: https://orcid.org/0000-0001-9737-8074
+  - name: Brian DeCost
+    affiliation: National Institute of Standards and Technology
+    orcid: https://orcid.org/0000-0002-3459-5888
+  - name: Lily Major
+    affiliation: Aberystwyth University, UK
+    orcid: https://orcid.org/0000-0002-5783-8432
+  - name: Keith Butler
+    affiliation: Rutherford Appleton Laboratory, UK
+    orcid: https://orcid.org/0000-0001-5432-5597
+  - name: Jeyan Thiyagalingam
+    affiliation: Rutherford Appleton Laboratory, UK
+    orcid: https://orcid.org/0000-0002-2167-1343
+  - name: Francesca Tavazza
+    affiliation: National Institute of Standards and Technology
+    orcid: https://orcid.org/0000-0002-5602-180X
+  - name: Philipp Benner
+    affiliation: Bundesanstalt für Materialforschung und -prüfung BAM
+    orcid: https://orcid.org/0000-0002-0912-8137
+    github: https://github.com/pbenner
+repo: https://github.com/usnistgov/alignn
+url: https://jarvis.nist.gov/jalignn
+doi: https://doi.org/10.1039/D2DD00096B
+preprint: https://arxiv.org/abs/2209.05554
+requirements:
+  ase: 3.22.0
+  dgl-cu111: 0.6.1
+  numpy: 1.24.3
+  pandas: 2.0.1
+  scikit-learn: 1.2.2
+  torch: 1.9.0+cu111
+trained_for_benchmark: false
```

Comment:
@pbenner Just to clarify, you used `alignnff_wt10`?

Reply:
Exactly, it was called best_model.pt before:
[alignn/ff/best_model.pt → alignn/ff/alignnff_wt10/best_model.pt](https://github.com/usnistgov/alignn/commit/461b35fe6e5ed7ade7cbf9b345773e941371ecfc#diff-621ef7421f3ee795c8e2e19a640fb679f900ce48dadfd962fef6176b34ca7258)

Reply:
Not sure if it makes sense to test some of the newly added models. Can give it a try, at least check if the convergence is better